packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -83,6 +98,39 @@ const prepareLocalChainFacadeKit = (
             this.facets.makeAccountWatcher,
           );
         },
+        /**
+         * Get asset info from agoricNames.vbankAsset.
+         *
+         * Caches the query to agoricNames in the first call. Subsequent lookups
+         * are via cache unless a refetch is specified.
+         *
+         * @param {boolean} [refetch] if true, will invalidate the cache
+         * @returns {Vow<AssetInfo[]>}
+         */
+        getVBankAssets(refetch) {
```

Comment:
As the jsdoc above says, this will "get asset info" not assets.
```suggestion
        getVBankAssetInfo(refetch) {
```